### DISCUSSION TITLE
DateTime without time / only data? It is possible?

### DISCUSSION THREAD

**Discussion by mxcdh** [prisma/prisma#12878] [Q&A] [answered]
Hello

I can in Prisma create columns without data, without time?

my model at the moment:
```
model modelName {
  id         Int      @id @default(autoincrement())
  createdAt  DateTime @default(now())
  lastNumber Int
}
```

Actual results:
```
2	2022-04-19 12:28:04.591+00	45
```

I want to generate records like this:
2	2022-04-19	45

**Comment by austincrim** [comment, marked as answer]
Hey @mxcdh 👋 ,

This is not currently possible, but we may revisit our `DateTime` handling in the future. If you really just want the time, you could store `createdAt` as a `String`, but you would lose some of the built-in `DateTime` features we offer, like `@default(now())`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
